pkg/entrypoint/entrypointer_test.go

```diff
@@ -160,6 +165,18 @@ func TestEntrypointer(t *testing.T) {
 	}, {
 		desc:                "breakpointOnFailure to wait or not to wait ",
 		breakpointOnFailure: true,
+	}, {
+		desc:       "Runner completes if not cancelled",
+		cancelFile: ".",
```

Comment:
Can we make this `cancelfile` or something? `"."` is an odd name for this file.

Reply:
Thank you for the review. Fixed.